src/main/resources/db/migration/mysql/V1.9.4__invite-table-status-index.sql

```diff
@@ -0,0 +1,2 @@
+create index invite_status_idx on `invite` (status);
+create index invite_for_email_idx on `invite` (for_email);
```

Comment:
DB Script file name does not reflect that the index is getting created in email column. Could you please either create two DB script files or rename the DB Script file to reflect both the indexes.

Reply:
Done. The DB file name has been changed now to include both column in the file name for which indexes are being added.